<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="pushButtonOpen">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>打开文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLex">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>0</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>词法分析</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonGram">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>0</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>语法分析</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSrc">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>源程序</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelLex">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>词法分析</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelGram">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>50</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>语法分析</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditSrc">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>580</width>
     <height>250</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditLex">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>580</width>
     <height>200</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditGram">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>100</y>
     <width>880</width>
     <height>400</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelMid">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>600</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>中间代码</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditMid">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>650</y>
     <width>580</width>
     <height>300</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonMid">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>0</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>中间代码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonTar">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>0</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>目标代码</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTar">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>500</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>目标代码</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditTar">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>550</y>
     <width>880</width>
     <height>400</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
